_docs/release-notes.md

```diff
@@ -351,6 +360,26 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.6.5 (Released 2018-07-24)
+It's another Bug Fix Bonanza™! This this round is devoted to supported Medtronic 5/7 pumps.  
+
+We fixed a bug for pumpers who use dual or square boluses without extending their bolus (100%/0%).  
```

Comment:
dual or square -> dual/square ? I think the "Medtronic term" requires use of the forward slash.

Reply:
I'll update this.